presto-docs/src/main/sphinx/release/release-0.229.rst

```diff
@@ -0,0 +1,30 @@
+=============
+Release 0.229
+=============
+
+General Changes
+_______________
+* Add geospatial function line_interpolate_point.
+* Improve hash value computation for structrual types to avoid collision.
```

Comment:
Optimizations should be before additions (fix > optimization > addition > deletion): https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#order-of-changes